services/workbench-bff/src/api/routes.py

```
@@ -1723,32 +1726,204 @@ async def generate_documentation(
+                        "tsx",
+                        os.path.join(
+                            repo_root, "libs/documentation/scripts/generate.ts"
+                        ),
+                        "--project",
+                        request.project_id,
+                        "--type",
+                        doc_type.lower(),
+                        "--output",
+                        tmp_dir,
+                    ]
+                    gen_result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)